integration/sandbox_cgroup/check_cgroups_sandbox.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This will enable the sandbox_cgroup_only
+# to true and verify the cgroups
+
+dir_path=$(dirname "$0")
+source "${dir_path}/../../lib/common.bash"
+source "${dir_path}/../../.ci/lib.sh"
+tests_repo="${tests_repo:-github.com/kata-containers/tests}"
+TEST_SANDBOX_CGROUP_ONLY="${TEST_SANDBOX_CGROUP_ONLY:-}"
+image="centos"
+container_name="test_cgroups"
+payload="tail -f /dev/null"
+
+if [ -z "${TEST_SANDBOX_CGROUP_ONLY}" ]; then
+	info "Skip: TEST_SANDBOX_CGROUP_ONLY variable is not set"
+	exit 0
+fi
+
+function setup() {
+	clean_env
+	check_processes
+}
+
+function check_cgroups_sandbox() {
+	pushd "${GOPATH}/src/${tests_repo}
+	".ci/toggle_sandbox_cgroup_only.sh" true
+	docker run -d --name "${container_name}" --runtime "${RUNTIME}" "${image}" sh -c "${payload}"
+	get_id=$(docker inspect --format='{{.Id}}' "${container_name}")
+	# Verify first cgroup
+	first_cgroup="a *.* rwm"
+	check_first_cgroup=$(grep -Fx "${first_cgroup}" /sys/fs/cgroup/devices/docker/kata_"${get_id}"/devices.list)
```

Comment:
This looks like a docker-specific test? If so, could you:

- make that clear in the commit.
- maybe renaming the test to include "docker"?
- add a comment at the top of the file explaining that the caller should only run this test if docker is available.
- maybe check that docker is available on the system and just `exit 0` (or `exit 1`?) if docker not available.
- consider creating a global variable like this:

  ```sh
  docker_cgroup_prefix="/sys/fs/cgroup/devices/docker"
  ```

  ... and then referring to that in the test functions.